haas/cli.py

```diff
@@ -43,12 +43,12 @@ class HTTPClient(object):
     __metaclass__ = abc.ABCMeta
 
     @abc.abstractmethod
-    def request(method, url, data=None):
+    def request(method, url, data=None, params=None):
         """Make an HTTP request
 
-        Makes an HTTP request on URL `url` with method `method` and request
-        body `data` (if supplied). May add authentication or other
-        backend-specific information to the request.
+        Makes an HTTP request on URL `url` with method `method`, request body
+        `data`(if supplied) and GET parameter `params`(if supplied). May add
```

Comment:
Should call them query parameters, not GET parameters; they've got nothing to do with GET per se, it's just that you can't have a body in GET so...

This is just the bit after the question mark in the URL, e.g. `https://example.com/foo/bar?baz=1&quux=hello`